tests/v2/validation/pipeline/Jenkinsfile.release.upgrade.local

```diff
@@ -172,7 +238,22 @@ node {
               stage("Build Environment") {
                 try {
                   sh "docker run -v ${validationVolume}:/root --name ${buildTestContainer} -t --env-file ${envFile} " +
-                  "${imageName} sh -c \"${workPath}pipeline/scripts/setup_environment.sh\""
+                  "${imageName} sh -c \"${workPath}pipeline/scripts/setup_environment.sh;${workPath}pipeline/scripts/build_qase_auto_testrun.sh\""
+
+                  sh "docker cp ${buildTestContainer}:${workPath}${qaseRun} ./"
+                  sh "./${qaseRun} --startRun=true"
+
+                  testRunID = sh(
+                      script: "./yq '.id'  ./${testRunConfig}",
+                      returnStdout: true
+                    ).trim()
+
+                  if (testRunID != "null" && testRunID != "") {
+                      echo "Extracted Test Run ID: '${testRunID}'"
+                  } else {
+                      echo "Failed to extract Test Run ID"
```

Comment:
Do we want to throw an error here also if it fails to extract the Test Run ID?

Reply:
yeah I added this otherwise if its returning null, the job is getting created with null id. and the results will be lost.